docs/AppService/GetActiveServiceConsent/index.md

```diff
@@ -0,0 +1,85 @@
+## GetActiveServiceConsent
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Request an app service to become the active service of it's type
+
+### Request
+
+!!! MUST
+1. Prompt user to activate service with `serviceID`
+2. Respond with the result from the user in the `activate` parameter
+!!!
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|serviceID|String|true||
+
+### Response
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|activate|Boolean|true||
+
+### Sequence Diagrams
+|||
+GetActiveServiceConsent
```

Comment:
This isn't actually used on the developer portal, I was just following convention with this, which is to just putting the RPC name here.

Reply:
I guess I just wanted the user to know which flow each diagram corresponds to. For one diagram it would probably be fine to just use the RPC name but for a case like this https://smartdevicelink.com/en/docs/pull_request/9bd751ee5004832309dc9411e80b2df508f445a7/appservice/getappservicedata/#sequence-diagrams it might be better for the user to see the (HMI Provider) or (HMI Consumer) before clicking view diagram 🤷‍♂️